pytest/action.yml

```diff
@@ -12,7 +12,7 @@ runs:
   - name: Get python version
     id: python_version
     shell: bash
-    run: echo "::set-output name=version::$(sed -n 's/^PYTHON_VERSION = \([0-9]\.[0-9]\).*/\1/p' Makefile || echo 3.10)"
+    run: echo "::set-output name=version::$(sed -n 's/^PYTHON_VERSION = \([0-9]\.[0-9]*\).*/\1/p' Makefile || echo 3.10)"
```

Comment:
Should be +, not *. Or better yet, {1,2}.

Reply:
Had trouble with the syntax, since it's not regex directly. But it turns out, you can escape the braces.